### PR TITLE
feat(heureka): switches to TopNavigationItem

### PR DESCRIPTION
**Pull request by hodanoori**
# Summary

Uses TopNavigationItem instead of TabNavigationItem as different views which are displayed in each tab are belonging to one application.

# Related Issues

- Issue 1:  https://github.com/cloudoperators/heureka/issues/218

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by andypf — DISMISSED** [review]

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 1]
```diff
@@ -37,15 +37,22 @@ const TabContext = () => {
...
-        <TabNavigationItem key={tab.value} icon={tab.icon} label={tab.label} value={tab.value} />
+        <TopNavigationItem
+          key={tab.value}
+          label={tab.label}
+          active={activeTab === tab.value} // Set the active item
+          onClick={() => setActiveTab(tab.value)} // Trigger tab change
+        />
       )),
-    []
+    [activeTab, setActiveTab]
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:51]: Tab -> Nav

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 2]
```diff
@@ -37,15 +37,22 @@ const TabContext = () => {
...
   )
 
-  const memoizedTabPanels = useMemo(
+  // Memorized tab panels
+  const memorizedTabPanels = useMemo(
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:55]: Tab -> Nav

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 3]
```diff
@@ -37,15 +37,22 @@ const TabContext = () => {
...
     () =>
       TAB_CONFIG.map((tab) => (
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:57]: Tab -> nav

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 4]
```diff
@@ -37,15 +37,22 @@ const TabContext = () => {
...
         <TabPanel key={tab.value} value={tab.value}>
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:58]: You shouldn't be using the tab panel anymore after the switch to TopNavigation

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 5]
```diff
@@ -57,10 +64,8 @@ const TabContext = () => {
 
   return (
     <>
-      <TabNavigation activeItem={activeTab} onActiveItemChange={(value) => setActiveTab(value)}>
-        {memoizedTabs}
-      </TabNavigation>
-      <Container py>{memoizedTabPanels}</Container>
+      <TopNavigation onActiveItemChange={() => {}}>{memorizedTabs}</TopNavigation>
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:67]: Tab -> Nav

File: apps/heureka/src/components/tabs/TabContext.jsx [hunk 6]
```diff
@@ -57,10 +64,8 @@ const TabContext = () => {
...
+      <Container py>{memorizedTabPanels}</Container>
```
Comment (edda) [inline apps/heureka/src/components/tabs/TabContext.jsx:68]: Tab -> Nav

File: apps/heureka/src/components/tabs/TabContext.jsx (hunk 4 above)
Comment (hodanoori) [reply apps/heureka/src/components/tabs/TabContext.jsx:58]: why? as the tabPanel contains the div for each top nav item content (its children) should be there. I can merge it into NavEntryContext but I rather prefer to keep it in separated control as it is now.

**Review by edda — APPROVED** [review]
I still think there's something wrong with the changeset and the `"@fake-scope/fake-pkg"` thing. But adding approval for the code change.

**Review by andypf — APPROVED** [review]

**Pull request merged**